### PR TITLE
User admin

### PR DESCRIPTION
**Pull request by Mgaren**
branch FAQ oublié de changer de branch mais c la user ADMIN

### CODE REVIEW

**Review by axelhamil — DISMISSED** [review]

**Review by axelhamil — DISMISSED** [review]

**Review by maxime-auneau — COMMENTED** [review]
ok

**Review by maxime-auneau — DISMISSED** [review]
**C'est ok le conflit est résolu**

**Review by leobro7 — DISMISSED** [review]

**Review by RomainClair — CHANGES_REQUESTED** [review]
The migration/Version files are not compatible with the dev branch ones.
The caterer table already exists
You shouldn't have unused or test code in your dev branch

**Review by axelhamil — DISMISSED** [review]

File: src/Controller/SessionController.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+     */
+    public function index(SessionInterface $session): Response
+    {
+        if (!$session->has('total')) {
+            $session->set('total', 0);
+        }
+        $session->get('total');
+        return $this->render('session/session.html.twig');
+    }
+}
```
Comment (RomainClair) [inline src/Controller/SessionController.php]: You should not merge test or incomplete code in the dev branch

**Review by RomainClair — CHANGES_REQUESTED** [review]
The migration files are still not compatible with  the dev ones.
It is impossible to apply the migrations from this branch on a database in the state it is after applying the migrations on the dev branch

**Review by leobro7 — DISMISSED** [review]

**Review by axelhamil — DISMISSED** [review]
It's ok for me, migrations work on my side

**Review by maxime-auneau — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**